projects/plugins/boost/app/modules/optimizations/cloud-css/Cloud_CSS.php

```diff
@@ -18,6 +18,11 @@
 
 class Cloud_CSS implements Pluggable, Has_Always_Available_Endpoints, Changes_Page_Output {
 
+	const REGENERATE_REASON_USER_REQUEST = 'user_request';
+	const REGENERATE_REASON_SAVE_POST    = 'save_post';
+	const REGENERATE_REASON_INVALIDATED  = 'invalidated';
+	const REGENERATE_REASON_FOLLOWUP     = 'followup';
```

Comment:
where can I read more about each reason and how they work? User request and save post are self explanatory, but "invalidated" and "followup" I'm not sure what triggers them and why, without searching throughout the codebase for them.

Reply:
Added comments